shell/platform/android/BUILD.gn

```diff
@@ -560,7 +560,7 @@ zip_bundle("android_symbols") {
   deps = [ ":flutter_shell_native" ]
 }
 
-zip_bundle("android") {
+zip_bundle("flutter_jar_zip") {
```

Comment:
do you know where artifacts.zip is used?

Reply:
It will be used by the recipe instead of creating its own `artifacts.zip`. See https://flutter.googlesource.com/recipes/+/refs/heads/master/recipes/engine.py#611